sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Grpc/Proto/settlement.proto

```diff
@@ -19,6 +20,18 @@ service Settlement {
 
   // Defers a message
   rpc Defer (DeferRequest) returns (google.protobuf.Empty) {}
+
+  // Get
+  rpc Get (GetRequest) returns (GetResponse) {}
+
+  // Set
+  rpc Set (SetRequest) returns (google.protobuf.Empty) {}
+
+  // Release
+  rpc Release (ReleaseSession) returns (google.protobuf.Empty) {}
+
+  // Renew
+  rpc Renew (RenewSessionLock) returns (RenewSessionLockResponse) {}
```

Comment:
Have we not had requests for RenewMessageLock as well? /cc @fabiocav

Reply:
We should absolutely try to keep the set of operations consistent between sessions/non-sessions where they overlap.   

+1 to adding RenewMessageLock, as I know that we've had some customers asking about it.

Reply:
Agreed. 

I'm not tracking active requests for that, but it's a matter of time, and as soon as this lands, any gaps will be highlighted.

Had a sync with @aishwaryabh and she is looking at the scope impact.